### ISSUE TITLE
Slash hack bug

### ISSUE THREAD

**Issue by klimashkin** [css/csso#159]
Some css hacks lead to unexpected results:

For example, such bootstrap class:
```
// Position radios and checkboxes better
input[type="radio"],
input[type="checkbox"] {
  margin: 4px 0 0;
  margin-top: 1px \9; /* IE8-9 */
  line-height: normal;
}
```
Becomes:
```
input[type="radio"],input[type="checkbox"]{margin:1px 0 0;line-height:normal}
```

**Comment by rinatio** [comment]
+1

**Comment by lahmatiy** [comment]
Fixed by #166
